core/lib/spree/testing_support/dummy_app.rb

```diff
@@ -12,14 +12,11 @@
 
 require 'solidus_core'
 
-RAILS_52_OR_ABOVE = Gem::Version.new(Rails.version) >= Gem::Version.new('5.2')
 RAILS_6_OR_ABOVE = Gem::Version.new(Rails.version) >= Gem::Version.new('6.0')
 
 # @private
 class ApplicationController < ActionController::Base
-  unless RAILS_52_OR_ABOVE
-    protect_from_forgery with: :exception
-  end
+  protect_from_forgery with: :exception
```

Comment:
This code was executed only when `RAILS_52_OR_ABOVE` was false, so I think the line should be removed

Reply:
You are right!